lib/src/Base/Func/openturns/ExpertMixture.hxx

```diff
@@ -47,7 +47,8 @@ public:
 
   /** Constructor */
   ExpertMixture(const Basis & experts,
-                const Classifier & classifier);
+                const Classifier & classifier,
+                const Bool supervised = true);
 
```

Comment:
It's really tied to the classifier here, so changing it in the RM is not enough, and will throw.